open_academy/__manifest__.py

```diff
@@ -17,7 +18,9 @@
         "views/open_academy_session_views.xml",
         "views/res_partner_views.xml",
         "wizards/open_academy_wizard_views.xml",
-        "views/open_academy_menus_views.xml"
+        "views/open_academy_menus_views.xml",
+        "views/open_academy_session_board.xml",
+        "reports/report.xml"
```

Comment:
Change file's name please.
```suggestion
        "reports/open_academy_session_report.xml"
```